flake.nix

```diff
@@ -228,7 +228,6 @@
         name = "fuel-core-dev";
         inputsFrom = with fuelpkgs; [
           fuel-core-nightly
-          fuel-gql-cli-nightly
```

Comment:
just checking why this is not needed anymore. Is this only used by older nightly builds and not required with newer builds?

Reply:
Yep exactly, from what I can tell it was renamed in December of 2022 ([commit](https://github.com/FuelLabs/fuel-core/commit/07a2888f2ae45bebab83c7219b717f3e30a1990e)). So with all of the nightlies from 2022 removed, there is nothing named `fuel-gql-cli-nightly` anymore, which results in [this error](https://github.com/FuelLabs/fuel.nix/actions/runs/13254691729/job/36999374947) "undefined variable 'fuel-gql-cli-nightly'"

I'm not 100% sure this is what's happening, just what I've gleaned from staring at this for a few hours.

Reply:
thanks, makes sense.